CHANGELOG.md

```diff
@@ -7,9 +7,14 @@ All notable changes to this project will be documented in this file.
 
 > # Next Release
 >
-> ### API Breaking
-> N/A
->
+### API Breaking
+ - **String** 
+    - `length` is deprecated, use native `count` instead.
+    - `slicing(i:)` is deprecated, use `string[safe: i]` instead.
+    - `slicing(from: to:)` is deprecated, use `string[safe: start..<end]`
+    - `firstIndex(of:)` is deprecated, use the natives `index(of: Character)` or `range(of: StringProtocol)` instead.
+    - `splitted(by:)` is deprecated,  use the native `split(separator: )` instead.
+    - `reversed() -> String` is deprecated, use the Swift 4 new `reversed() -> ReversedCollection<String>` 
```

Comment:
@LucianoPAlmeida Please add the PR number as a reference under this part of the CHANGELOG

Reply:
Added :)